sharding/proposer/service.go

```diff
@@ -80,8 +83,8 @@ func (p *Proposer) proposeCollations() {
 		case <-p.ctx.Done():
 			log.Error("Proposer context closed, exiting goroutine")
 			return
-		case err := <-p.txpoolSub.Err():
-			log.Error(fmt.Sprintf("Subscriber closed: %v", err))
+		case <-p.txpoolSub.Err():
+			log.Error("Subscriber closed")
```

Comment:
Is this really an error level message? 
Same comment for line 84. 

I think it should be info or debug.